skbio/metadata/_intersection.pyx

```diff
@@ -0,0 +1,513 @@
+"""
```

Comment:
Please add to this doc string at the top:

- where this file came from (e.g., link to the BxPython repo, and the date that you accessed it)
- list of the modifications that you made to it, if any